src/Config.cpp

```diff
@@ -77,6 +78,30 @@ void Config::readFromFile(const string &file) {
     }
 }
 
+void Config::readFromMap(map<string, any> &mp) {
+    bool given = false;
+    string CTL = "column_table_limit";
+    string CL = "column_limit";
+    for (auto kv : mp) {
+        auto key = kv.first;
+        if (node_[key]) {
+            if (*((kv.second).type().name()) == 'i') {
```

Comment:
Looks strange. Why not just write `kv.second.type().name() == "int"`

Reply:
If we not use *, it shows 
`ISO C++ forbids comparison between pointer and character`
And, `*((kv.second).type().name())` it gives character as 'i' for int, 'c' for character, 'b' for bool so on.

Reply:
Convert to std::string or use strcmp

Reply:
Done.